storage/storage_interface.ml

```diff
@@ -266,7 +271,7 @@ module SR = struct
 	(** Functions which manipulate SRs *)
 
 	(** [create dbg sr name_label name_description device_config physical_size] creates an sr with id [sr] *)
-	external create : dbg:debug_info -> sr:sr -> name_label:string -> name_description:string -> device_config:(string * string) list -> physical_size:int64 -> unit = ""
+	external create : dbg:debug_info -> sr:sr -> name_label:string -> name_description:string -> device_config:(string * string) list -> physical_size:int64 -> (string * string) list = ""
```

Comment:
So it returns the UUID as a string * string list? 😮

Reply:
It returns an updated device-config, containing the UUID. I think it is the equivalent of  `{ foo with uuid = ...}`, except device-config is a string map not a regular type.